packages/benchmarks/generator/package.json

```diff
@@ -40,6 +40,7 @@
   },
   "devDependencies": {
     "puppeteer": "^15.4.0",
-    "tachometer": "^0.7.0"
+    "tachometer": "^0.7.0",
+    "typescript": "~4.7.4"
```

Comment:
No action required – just noting for myself to fix this in the 3.0 branch. Should be TS 5.0